### PR TITLE
feat: SDK relay launch script options

### PR DESCRIPTION
**Pull request by typotter**
## Motivation and Context
Each SDK relay requires a build process to be executed in an environment configured to do so.

## Changes overview
- Specify a platform-specific build-and-run script
- use `docker-run.sh` if available
- properly exit runner script with code

### CODE REVIEW

File: package-testing/sdk-test-runner/test-sdk.sh [hunk 1]
```diff
@@ -139,6 +155,7 @@ case "$command" in
         echo "  ... Starting Test Cluster node [Eppo-exp/sdk-test-runner]"
         
         docker run \
+          --add-host host.docker.internal:host-gateway \
```
Comment (typotter) [inline package-testing/sdk-test-runner/test-sdk.sh:158]: docker on github actions is a little bit different. This is one of those gotcha differences

File: package-testing/sdk-test-runner/test-sdk.sh [hunk 2]
```diff
@@ -67,6 +70,7 @@ export SDK_RELAY_PORT="${SDK_RELAY_PORT:-4000}"
 export EPPO_SCENARIO_FILE="${EPPO_SCENARIO_FILE:-scenarios.json}"
 export EPPO_TEST_DATA_PATH="${EPPO_TEST_DATA_PATH:-./test-data}"
 
+PLATFORM="${PLATFORM:-linux}"
```
Comment (leoromanovsky) [inline package-testing/sdk-test-runner/test-sdk.sh:73]: Did you consider no default? I know it is tedious to add but could add clarity to where it is run

**Review by leoromanovsky — APPROVED** [review]
Kudos for figuring out the mechanics for how this operates on GH

File: package-testing/sdk-test-runner/test-sdk.sh (hunk 2 above)
Comment (typotter) [reply package-testing/sdk-test-runner/test-sdk.sh:73]: Ya, it would be best to be explicit here.

**Pull request merged**